scripts/restore_backup_osgp_dbs.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+HOST=localhost
+PORT=5432
+USER=osp_admin
+BACKUP_DIR=$HOME/backups/osgp
+LOGFILE=restore.log
+
+if [ $# -eq 0 ];then
+    echo "The first argument must be a backup (file from $BACKUP_DIR/) ..."
+	ls $BACKUP_DIR/
+    exit
+fi
+
+BACKUP_DIR=$BACKUP_DIR/$1
+
+function restore() {
+        echo "- restoring $BACKUP_DIR/$1.gz ..."
+	dropdb -h $HOST -p $PORT -U $USER --if-exists $1 >> $LOGFILE
+	createdb -h $HOST -p $PORT -U $USER $1 >> $LOGFILE
+	cat $BACKUP_DIR/$1.gz | gunzip | psql -U $USER $1 >> $LOGFILE
+}
+
+mkdir -p $BACKUP_DIR
+
+# PGAdmin may not be running.
+killall pgadmin3
```

Comment:
It would be nicer for users of pgAdmin 3 to test if the pgAdmin 3 process is running and abort the script instead of just killing an application like this.
Consider to revise this approach.